ui/arduino2/views/components/file-list.js

```diff
@@ -1,5 +1,15 @@
-function DiskFileList(state, emit) {
+function onKeyEvent(e) {
+  if(e.key.toLowerCase() === 'enter') {
+    e.target.blur()
+  }
+  if(e.key.toLowerCase() === 'escape') {
+    e.target.value = null
+    e.target.blur()
+  }
+}
+
```

Comment:
I would define this function inside `DiskFileList` and `BoardFileList` so we it's not declared on the global scope.
The repetition is annoying but we should solve it in a separate task, I think.